test/e2e/secret_test.go

```diff
@@ -336,6 +336,16 @@ var _ = Describe("Podman secret", func() {
 		Expect(session.OutputToStringArray()).To(HaveLen(1))
 	})
 
+	It("podman secret rm --ignore", func() {
+		remove := podmanTest.Podman([]string{"secret", "rm", "non-existent-secret"})
+		remove.WaitWithDefaultTimeout()
+		Expect(remove.ErrorToString()).To(Equal("Error: no secret with name or id \"non-existent-secret\": no such secret"))
```

Comment:
Oops! I'm sorry, I meant that string checks should be done _in addition to_, not instead of, exit-status. They are both important. Could you please reinstate the `Should...Exit()` checks? I'm so sorry for not being clear.